server/models/postgis/task.py

```diff
@@ -105,8 +110,18 @@ def get_last_status(project_id: int, task_id: int):
                     TaskHistory.action == TaskAction.STATE_CHANGE.name) \
             .order_by(TaskHistory.action_date.desc()).first()
 
+        if result == None:
+            return TaskStatus.READY
+
         return TaskStatus[result[0]]
 
+    @staticmethod
+    def get_last_action(project_id: int, task_id: int):
```

Comment:
Can this method be removed?  Acutally I see this is subtley different from method above, might be worth expaning the comment here to explain